Cargo.toml

```diff
@@ -28,6 +28,9 @@ path = "src/lib.rs"
 native-tls = "0.1"
 regex = "0.2"
 bufstream = "0.1"
+#imap-proto = "0.1"
+imap-proto = { git = "https://github.com/djc/imap-proto.git" }
```

Comment:
Why does this use the git repo for this dependency instead of the one at crates.io ?

Reply:
Ah, the required changes from `imap-proto` have since been released. Removed in e687e852fe9c75605cde45d7ed68961920e51d54